modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -0,0 +1,402 @@
+= Anypoint Runtime Fabric Installation Prerequisites
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: install-port-reqs.adoc, install-sys-reqs.adoc
+
+Review the following requirements with your operations and IT teams to ensure that all prerequisites are met before installing Anypoint Runtime Fabric. The Runtime Fabric installation process fails if the minimum requirements are not met.
+
+If needed, contact your MuleSoft representative for assistance.
+ 
+See xref:architecture.adoc[Runtime Fabric Architecture] for a description of development and production configurations.
+
+== General Knowledge
+Knowledge in the following areas is required to perform a Runtime Fabric installation:
+
+* Linux system commands
+* Working in Linux with root access. Installation requires root access (permission to run the `sudo` command) on all servers in your environment
```

Comment:
GLOBALLY fix list items to not exceed one sentence, per the style guide. I have commented about this many times.
```suggestion
* Working in Linux with root access
+
+ Installation requires root access (permission to run the `sudo` command) on all servers in your environment.
```
Why do we need the second sentence?